pyupgrade.py

```diff
@@ -819,17 +883,17 @@ def _fix_percent_format_dict(tokens, start, node):
 
     key_indices = []
     for i, token in enumerate(tokens[brace:brace_end], brace):
-        k = keys.pop(token.offset, None)
-        if k is None:
+        key = keys.pop(token.offset, None)  # type: Optional[ast.Str]
```

Comment:
I'm surprised this requires an annotation 🤔

Reply:
it does not, the rename was enough